expr/builder_test.go

```diff
@@ -17,6 +17,7 @@ func TestExprBuilder(t *testing.T) {
 		Reg:        expr.NewEmptyExtensionRegistry(&extensions.DefaultCollection),
 		BaseSchema: &boringSchema.Struct,
 	}
+	precomputedExpression, _ := expr.NewLiteral(int32(3), false)
```

Comment:
should we use examples that aren't Literals since wrap methods and such for literals, such as using the builder to create an expression first and then using that expression later, etc.

Reply:
I was pondering merging Wrap and WrapExpression into Wrap (with the functionality of WrapExpression) since they are essentially the same.

I will also add a non-literal Expression example.